app/dashboard/templates/profile_details.html

```diff
@@ -1,248 +1,210 @@
-{% load i18n email_obfuscator %}
-{% load humanize %}
-{% load static %}
+{% load i18n email_obfuscator %} {% load humanize %} {% load static %}
 <!DOCTYPE html>
 <html lang="en">
 
-<head>
-    {% include 'shared/head.html' %}
-    {% include 'shared/cards_pic.html' %}
-</head>
-<body class="interior {{active}}">
+  <head>
+    {% include 'shared/head.html' %} {% include 'shared/cards_pic.html' %}
```

Comment:
As @thelostone-mc was saying, can we drop the second `{% include %}` to it's own line?